File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/model/dto/DictionaryQo.java
package com.github.liulus.yurt.system.model.dto;

import com.github.liulus.yurt.convention.data.PageQuery;
import lombok.Data;

/**
 * User : liulu
 * Date : 2017/4/8 20:52
 * version $Id: DictionaryVo.java, v 0.1 Exp $
 */
@Data
public class DictionaryQo extends PageQuery {

    private static final long serialVersionUID = -2144418218430870185L;

    private String keyword;

    private Long parentId;

}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/model/dto/DictionaryQo.kt
package com.github.liulus.yurt.system.model.dto

import com.github.liulus.yurt.convention.data.PageQuery

/**
 * User : liulu
 * Date : 2017/4/8 20:52
 * version $Id: DictionaryVo.java, v 0.1 Exp $
 */
class DictionaryQo : PageQuery() {
    var keyword: String? = null
    var parentId: Long? = null

}